lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -22,13 +21,117 @@ function distribute(node) {
     return distributeAndSimplifyMultiplication(node);
   }
   else if (Node.Type.isOperator(node, '^')) {
+    /*
+      If it is a power node
+      1) If the exponent is negative, reciprocate the base
+      2) If the base is a multiplication node, distribute the exponent
+      3) Expand the base
+
+      e.g. (x y)^-1 -> 1/(x y)
+      e.g. (x y)^2 -> x^2 y^2
+      e.g. (x + 1)^2 -> (x + 1) (x + 1)
+    */
+
+    const [base, exponent] = node.args;
+
+    if (Negative.isNegative(exponent)
+        || (exponent.content && Negative.isNegative(exponent.content))) {
+      return reciprocateTerm(node);
+    }
+    else if (base
+        && Node.Type.isParenthesis(base)
+        && Node.Type.isOperator(base.content, '*')) {
+      return distributeExponent(node);
+    }
     return expandBase(node);
   }
   else {
     return Node.Status.noChange(node);
   }
 }
 
+/*
+  Given a node raised to a negative power, negate the exponent and reciprocate
+  the node. Returns a nodeChanged Status object.
+
+  e.g. (x y)^-1 -> 1/(x y)
+  e.g. (2x^2)^-1 -> 1/(2x^2)
+*/
+function reciprocateTerm(node) {
+  const exponent = node.args[1];
+  // Negate the exponent
+  node.args[1] = Node.Type.isParenthesis(exponent)
+    ? Node.Creator.parenthesis(Negative.negate(exponent.content))
+    : Negative.negate(exponent);
+
+  const one = Node.Creator.constant(1);
+
+  const newNode = Node.Creator.operator('/', [one, node]);
+
+  return Node.Status.nodeChanged(
+    ChangeTypes.NEGATIVE_EXPONENT, node, newNode, false);
+}
+
+
+/*
+  Given an expression in the form (base)^exponent, where base is a multiplication node
+  and the exponent is any term, we want to distribute the exponent over each term.
+  Returns a nodeChanged Status object.
+
+  e.g. (x^2 y^2)^(2) -> x^4 y^4
+  e.g. ((x + 1) (x + 2))^2 -> (x + 1)^2 (x + 2)^2
+*/
+function distributeExponent(node) {
+  // Base is required to be a multiplication of terms
+  const base = Node.Type.isParenthesis(node.args[0])
+        ? node.args[0].content
+        : node.args[0];
+
+  // Exponent can be any term: polynomial, constant, fraction, etc
+  const outerExponent = Node.Type.isParenthesis(node.args[1])
+        ? node.args[1].content
+        : node.args[1];
+
+  // Iterate over each factor in the base and multiply the outside exponent
+  // by the exponent of the term (implicit is one)
+  // e.g. (x^2 y^2)^2 -> x^(2 * 2) y^(2 * 2)
+  const newTerms = base.args.map(function(term) {
+    // Recurses when a term is a multiplication node
+    // e.g. (2x^2 * y)^2 -> 2^2 * x^(2 * 2) * y^2
+    if (Node.Type.isOperator(term, '*')) {
+      const newTerm = Node.Creator.operator('^', [term, outerExponent]);
+
+      return distributeExponent(newTerm).newNode;
+    }
+
+    // e.g. innerBase for x is x, 2 is 2, (x + 1) is (x + 1)
```

Comment:
Addressing the second part of your comment, I think we want to show that when we distribute exponents, we can multiply the outer and inner exponents for pedagogy reasons.